### PR TITLE
Update nothirdparty

### PR DESCRIPTION
**Pull request by gallegonovato**

### CODE REVIEW

File: nothirdparty [hunk 1]
```diff
@@ -114,10 +114,804 @@ media-amazon.com
...
+ws.amazon.com
+ws.amazon-adsystems.com
+z-eu.amazon-adsystem.com
+z-fe.amazon-adsystem.com
+z-in.amazon-adsystem.com
+z-na.amazon-adsystem.com
+z-na.amazon-adsystems.com
+acsechocaptiveportal.com
+www-cdn.jtvnw.net
+http://www.twitch.tv/
```
Comment (nickspaargaren) [inline nothirdparty:271]: Please remove http:// and extra / from domains

File: nothirdparty [hunk 2]
```diff
@@ -114,10 +114,804 @@ media-amazon.com
...
+mas-sdk.amazon.com
+# required for apps to be installed
+#mas-ext.amazon.com
+aax-eu.amazon-adsystem.com
+msh.amazon.co.uk
+# required for app installation / updates
+# amazonadsi-a.akamaihd.net
+mobileanalytics.us-east-1.amazonaws.com
+# alexa
+#cortana-gateway.amazon.com
```
Comment (nickspaargaren) [inline nothirdparty:880]: Please remove # in front of the domains, otherwise the domains will be skipped.

File: nothirdparty (hunk 1 above)
Comment (gallegonovato) [reply nothirdparty]: Done :) HAPPY NEW YEAR

File: nothirdparty (hunk 2 above)
Comment (gallegonovato) [reply nothirdparty]: Done :) HAPPY NEW YEAR

**Pull request merged**